### ISSUE TITLE
Ang Diaryo's metrics importer picking up BBG stats

### ISSUE THREAD

**Issue by jacqui** [news-catalyst/next-tinynewsdemo#977]
https://github.com/news-catalyst/next-tinynewsdemo/runs/4205061677?check_suite_focus=true

<img width="990" alt="Screen Shot 2021-11-15 at 1 31 44 pm" src="https://user-images.githubusercontent.com/3397/141713077-9f7f4354-48ff-45a9-a143-f4cf87ed2ef5.png">

**Comment by jacqui** [comment]
So far the tracking ID and view ID were wrong in Vercel. 

Ang Diaryo's GA settings should be:

* Tracking ID: 166777432-51
* View ID: 254158738

I'm going to create an .env file for Ang Diaryo now and add it to the tnc-org-envs repo. I've updated the GH Action environment settings for the data importer and the Vercel env settings. I'll redeploy, clear out any old analytics data, and re-run the data importer now.

**Comment by jacqui** [comment]
There was a problem with both the view ID in the importer and the tracking ID in vercel. I've fixed both of these, and I've added the environment settings for Ang Diaryo to [the tnc-org-envs repo](https://github.com/news-catalyst/tnc-org-envs/pull/3). I re-ran the data importer and, as I expected, no data was found for this yet-to-be-launched site. It's pulling the right analytics data, in other words.
